src/app/Core/ExportServices/modifyTableDataString.service.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * @desc escape all quotes and replaces "undefined" with an empty string
+ * @param {string} inputString - string to be modified
+ * @returns {string} - modified string
+ */
+export function modifyTableDataStringService(inputString: string): string {
+    return `"${inputString.replace(/"/g, '\\"').replace("undefined", "")}"`;

```

Comment:
why not return inputString.replace(...).replace(...)

Reply:
It is necessary to include commas within the filter values and not splitting them in the csv file.

Reply:
ok, i see, so the output is additionally surrounded by quotes? perhaps you should add this to the description of the function.

Reply:
Done. "Rename & Simplify"